### PR TITLE
Add --dry-run flag

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

**Comment by jazzdan** [comment]
See the first commit, the rest is just formatting.

File: index.js [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-let syncParser = subparsers.addParser('sync', {
+let syncParser = subparsers.addParser("sync", {
   addHelp: true,
-  help: 'sync an issue from github to clubhouse',
+  help: "sync an issue from github to clubhouse",
 })
 
-syncParser.addArgument(['-i', '--issue'], {
-  help: 'The github issue number to import into clubhouse',
+syncParser.addArgument(["-i", "--issue"], {
```
Comment (maiamcc) [inline index.js:27]: (this doesn't have to change now but) if the only use of this tool is to import an issue, shouldn't this just be a required positional arg and not a flag?

File: index.js [hunk 2]
```diff
@@ -87,32 +97,47 @@ function createClubhouseStoryForGithubIssue(issue) {
...
+    let storyPromise = findClubhouseStoryForGithubIssue(issue)
+    return Promise.all([issue, storyPromise])
+  })
+  .then(([issue, existingStory]) => {
+    if (existingStory) {
+      console.log("Found existing clubhouse issue:\n" + existingStory.app_url)
+    } else {
+      return createClubhouseStoryForGithubIssue(issue).then(response => {
+        if (isDryRun) {
+          console.log("Would create new clubhouse issue:\n")
```
Comment (maiamcc) [inline index.js]: consider setting this line apart visually and/or adding another line below to the effect of "🤐no changes made" so it's obvious at a glance whether we changed state or not. (Visually this looks v. similar to the "create an issue" message.)

**Review by maiamcc — APPROVED** [review]
noice!

**Review by nicks — APPROVED** [review]

**Pull request merged**